incubator/hnc/internal/validators/hncconfig.go

```diff
@@ -158,13 +160,17 @@ func (c *HNCConfig) checkConflictsForTree(gvk schema.GroupVersionKind, ao ancest
 	objs := ao.copy()
 	for _, o := range ns.GetSourceObjects(gvk) {
 		onm := o.GetName()
-		objs.add(onm, ns)
-		// If there are more than just the one (current) namespace we just added,
-		// there may be conflicts.
-		if objs.hasConflict(onm) {
-			// Currently the top conflicting ancestor would overwrite the object.
-			// TODO: check if this is a real conflict
-			conflicts = append(conflicts, fmt.Sprintf("  Object %q in namespace %q would overwrite the one in %q", onm, objs.top(onm), ns.Name()))
+		nsLabels := ns.GetLabels()
+		existingNs := objs[onm]
+		objs.add(onm, ns, nsLabels)
+		// Currently the top conflicting ancestor would overwrite the object.
+		// TODO: check if this is a real conflict
```

Comment:
Can you be more specific about this?

Reply:
Hm I didn't leave this TODO here, @yiqigao217 ?

Reply:
Any update on this one?

Reply:
Oh this TODO can be removed. since it's fixed by this PR.